### PR TITLE
Small refactor changes to be more react and typescript-friendly

### PR DESCRIPTION
**Pull request by edumorlom**
In an effort to make the website use types and be more react friendly, I have made some small refactoring changes.

1. function() is unbinded, to remove redundant code, I converted all functions to () => {} which is binded.
2. Added types to the State.
3. Added types to return functions.
4. Converted breadcrumb to use React Component instead of .innerHTML.
5. The map is a bit smaller. I divide by 3.5 instead 3.0. The map's legend was going out of boundaries.
6. Ran linter.
7. Adds public to methods as per Codacy recommendation.
8. Fixes any comments on previous PR.

Screenshot:
Screenshot 2020-09-10 at 7 57 18 PM

### CODE REVIEW

File: static/js/tools/choropleth/choropleth.tsx [hunk 1]
```diff
@@ -105,23 +127,23 @@ class ChoroplethMap extends Component {
           "statistical variable choice! Please select a new variable.";
       }
     );
-  }
+  };
 
   /**
-   * Set Per Capita.
+   * Set Per Capita on the State.
```
Comment (jeffreyoldham) [inline static/js/tools/choropleth/choropleth.tsx:133]:
```suggestion
   * Set per capita on the state.
```

**Review by jeffreyoldham — COMMENTED** [review]
Thanks.

**Review by beets — APPROVED** [review]
thanks for moving this to a separate PR! we can merge after you address jeffrey's comment

**Comment by edumorlom** [comment]
Ready to merge!

**Review by beets — APPROVED** [review]

**Comment by beets** [comment]
thanks for the quick turnaround!

**Pull request merged**